docs/index.md

```diff
@@ -20,6 +20,8 @@ This is the center for eZ Platform documentation for developers and system admin
 
 **Need Release Notes & older versions? See [Releases](releases/releases.md)**
 
+**Looking for eZ Publish (Legacy, from version 3.x to 5.x.) documentation? See [Legacy documentation](https://doc.ez.no/eZ-Publish)**
+
```

Comment:
This URL works for eZ Publish 3.x to 4.x. 
eZ Publish Platform (5.x) documentation is here: https://doc.ez.no/display/EZP/eZ+Publish+5.x+Developer+Documentation

Reply:
You are right, we probably need two links, one for eZ Publish Legacy and one for eZ Publish 5.